src/NServiceBus.Metrics/CriticalTime/CriticalTimeMetricBuilder.cs

```diff
@@ -23,6 +35,6 @@ public override void WireUp(FeatureConfigurationContext featureConfigurationCont
         });
     }
 
-    [Timer("Critical Time", "Messages", "Age of the oldest message in the queue.")]
+    [Timer("Critical Time", "Messages", "The time it took from sending to processing the message.")]
```

Comment:
Was preserved with the old crappy text because the description is baked into the counter and the scripts. I didn't dare to touch it. Maybe I was just not brave enough. I haven't checked deeper